Makefile

```diff
@@ -145,6 +148,6 @@ test-e2e-1.26: container-build-single-platform install-tools
 test-e2e-1.25: container-build-single-platform install-tools
```

Comment:
please remove the test-e2e-1.25 & test-e2e-1.26 make commands as they are no longer used in testing